src/librustc_session/session.rs

```diff
@@ -235,7 +235,37 @@ impl Session {
     }
 
     pub fn local_crate_disambiguator(&self) -> CrateDisambiguator {
-        *self.crate_disambiguator.get()
+        self.crate_disambiguator.get().copied().unwrap()
+    }
+
+    #[inline]
```

Comment:
Is there a reason these are `#[inline]`? I imagine there's no way they're hot enough to justify that without benchmarking and this will bump compile times up a bit since they'll be codegen'd into every module that uses them...

Reply:
No good one. I'll remove them.